src/s2i/bin/run

```diff
@@ -26,6 +26,12 @@ else
   exit 3
 fi
 
+if [ ${PLATFORM} == "el7" ]; then
+    VARNISH_VCL=/etc/opt/rh/rh-varnish{{ spec.version }}/varnish/default.vcl
```

Comment:
Wouldn't it be better to put this path into `multispec.yml` and then define `VARNISH_VCL` in the Dockerfile? I'm not that experienced with dist-gen, so not sure entirely about this.

Reply:
Makes sense. I moved the definition of `VARNISH_VCL` to Dockerfiles.